COMPATIBILITY.md

```diff
@@ -3,3 +3,4 @@
 | 1 (Testnet 70)            | v0.70.x                | v0.37.5  |   v1     |
 | 2 (Testnet 73)            | v0.73.x                | v0.37.5  |   v1     |
 | 3 (Testnet 74)            | v0.74.x                | v0.37.5  |   v1     |
+| 4 (Testnet 75)            | v0.75.x                | v0.37.5  |   v1     |
```

Comment:
Bumping crate version only in the COMPAT doc for now. Post-merge, we'll bump-and-tag directly on the main branch, to trigger the release.